server/mqtt_test.go

```diff
@@ -216,16 +216,14 @@ func testMQTTDefaultOptions() *Options {
 
 func testMQTTRunServer(t testing.TB, o *Options) *Server {
 	t.Helper()
-	o.NoLog = false
+	o.NoLog = true
```

Comment:
This was on purpose that I would set logging (NoLog=false) and a dummy logger below. The idea is to have better code coverage and test of debug/trace statements. Any reason you have removed this?

Reply:
Since I find myself changing this value every time I need to debug a unit test, I instinctively changed that. Will undo this change.